### PR TITLE
Running rubocop command

### PR DESCRIPTION
**Pull request by aleksgorbenko**
38 errors still left - most of them are just "the line is too long"

### CODE REVIEW

**Comment by lasley** [comment]
FYI the fact that there are errors and the build is passing means this isn't rigged up correctly.

Hint, take a look at your Rakefile

File: Rakefile [hunk 1]
```diff
@@ -4,6 +4,5 @@
 require File.expand_path('../config/application', __FILE__)
 require 'coveralls/rake/task'
 
-
```
Comment (aleksgorbenko) [inline Rakefile]: @lasley not too sure what is wrong here, I tried switching lines of code here, but it made no difference. I still get 38 offenses, most of which are pointing that the comments in the files are too long or absence of comments at the top of some modules. Not sure if I understood what else I would need to fix? Delete these comments from files, just shorten them, or?

**Comment by aleksgorbenko** [comment]
Ok, resolved conflicts on the branch. I am not sure @lasley if i did what you implied me to do, please let me know.

For now i will remove WIP tag.

**Comment by chadbaum** [comment]
:+1:

**Comment by lasley** [comment]
I'm pretty sure the CI isn't running Rubocop. You should have something like this in rakefile:

```
task test: :rubocop

task :rubocop do
  sh 'rubocop'
end
```

**Comment by lasley** [comment]
I lied, didn't realize they made this more streamlined in newer versions. My bad!

**Comment by lasley** [comment]
I just killed the conflicts in order to get this off the block. It's basically going to be one giant conflict the longer it sits, so needed to happen 😉

**Comment by aleksgorbenko** [comment]
Ah thanks @lasley ! by "killing the conflicts" you mean you solved them, or made Rubocop force ignore them?

**Comment by lasley** [comment]
Solved them. Ignoring things is never good 😉

**Comment by aleksgorbenko** [comment]
Alright, Thanks! So just to confirm, you just shortened the lines of all the comments that had trouble passing the RuboCop check?

**Comment by lasley** [comment]
Nope, and if the build was passing with failures then they did not streamline this in the way that they supposedly did. My Rakefile comment still applies then.

When I initially looked at the build and it seemed like Rubocop ran, but now that I look further it was just the branch naming & it is not running. We'll need a ticket to fix the implementation.

**Comment by chadbaum** [comment]
I'll put a ticket on the board.  Aleks -- feel free to assign it to yourself if you want another stab at it.

**Pull request merged**